index.md

```diff
@@ -103,6 +103,20 @@ workshop is only open to people from a particular institution.
 {% include lc/who.html %}
 {% endif %}
 
+{% comment %}
+HUMAN LANGUAGE
+{% endcomment %}
+
+{% unless site.human_language == "" %}
+<p id="human-language">
+  <strong>The workshop will be conducted in {{ site.human_language }}.</strong>
+  For this course, we will expect our participants to have basic understanding
+  of {{ site.human_language }}. We will have several helpers to assist you if
+  you need assistance with understanding the lesson material.
```

Comment:
I think the "with" in the last line does not need to be here. 

I do still find this somewhat confusing. Is the idea that the helpers will be fluent in both the workshop language and another language to help with translation? 

Maybe there should be two fields in config (I don't love this proposal, but thinking together is better than alone):
`workshop-human-language` and `local-human-language` and then display the paragraph if those do not match (with appropriate update to the comment)? and in the reference to helpers can note that the helpers will be competent in both workshop and local languages? @malvikasharan @raynamharris would that be helpful?

Reply:
yes, I think that the intent was that helpers would be proficient both language and could help with translation if needed. That sentence I took from @malvikasharan's workshop might be too specific to be generalizable to all workshops, and it might be best to leave it out?